test/request-opts.js

```diff
@@ -0,0 +1,71 @@
+import assert from 'assert';
+import { Client } from '../lib';
+import nock from 'nock';
+import sinon from 'sinon';
+
+describe('request-opts', () => {
```

Comment:
Would be good to swap the arrows functions out for regular functions. Using arrow functions with Mocha is discouraged: https://mochajs.org/#arrow-functions

Reply:
I totally agree, but I was just trying to keep it consistent with the rest of the test suite - most people prefer that! Shall I change or?

Reply:
I think it'd be good to change it. We shouldn't be doing this elsewhere in the codebase

Reply:
Nice! Thanks. Yep, the other tests are definitely for a separate PR